From 97c0fc0aeaf7fc96b3df8ba773d1be3c13c9c24c Mon Sep 17 00:00:00 2001
From: Gayathri Athreya <gathreya@email.arizona.edu>
Date: Fri, 5 Jun 2015 12:12:43 -0700
Subject: [PATCH] RESKC-418:Warning should not count as rules errors. In
 Kualico instance: Negative under recovered F&A generated in budget; values do
 not populate to Commitments> UR Distribution. Cannot resolve validation.

Create a proposal type Research.
Create a budget;
in Settings: select Rate Type of TDC and select Unrecovered Rate Type of FUNSN
in Rates screen: set TDC rate to 10%
Add budget expenses and generate periods
Review calculated Rate in budget item Details modal: TDC rate calculates correctly.
Review the P&T screen: note negative UR amount in column
Complete Budget action prompts Validation Error to clear audit.
Open Commitments> Unrecovered F&A screen. System says no UR generated.
---
 .../coeus/common/budget/impl/nonpersonnel/BudgetExpensesAuditRule.java   | 1 -
 1 file changed, 1 deletion(-)

diff --git a/coeus-impl/src/main/java/org/kuali/coeus/common/budget/impl/nonpersonnel/BudgetExpensesAuditRule.java b/coeus-impl/src/main/java/org/kuali/coeus/common/budget/impl/nonpersonnel/BudgetExpensesAuditRule.java
index 9d0333a..6073590 100644
--- a/coeus-impl/src/main/java/org/kuali/coeus/common/budget/impl/nonpersonnel/BudgetExpensesAuditRule.java
+++ b/coeus-impl/src/main/java/org/kuali/coeus/common/budget/impl/nonpersonnel/BudgetExpensesAuditRule.java
@@ -202,7 +202,6 @@ protected boolean verifyUnderRecoveryAmount(BudgetLineItem budgetLineItem, int b
             auditErrors.add(new AuditError(budgetNonPersonnelRule.getPageId(), 
             		KeyConstants.WARNING_UNRECOVERED_FA_NEGATIVE, budgetNonPersonnelRule.getPageId(), 
             		new String[]{additionalMessage}));
-            return false;
        }
         return true;
 	}
